### PR TITLE
Meta Description Fallback

### LINKED ISSUE #1430: Meta description fallback
**Is your feature request related to a problem? Please describe.**
The value of `promo.summary` in the data is currently empty. We are waiting on the data team to add this field https://github.com/bbc/simorgh/issues/842
Since we are using this value for the meta description tag, we have an empty tag.  This is not good for SEO.  This reduces our lighthouse SEO score from 80 to 70.  

When running Lighthouse for a page, `/news/articles/c85pqyj5m2ko`, there is an error about the lack of description text.

Screen Shot lighthouse

**Describe the solution you'd like**
Ensure there is a fallback of the meta description element of `promo.headlines.seoHeadline` if the `promo.summary` is not provided. 

- [x] Initially labelled with ["Bug"](https://github.com/bbc/simorgh/labels/Bug)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #1430

When running Lighthouse for a page, `/news/articles/c85pqyj5m2ko`, there is an error about the lack of description text.

Screen Shot lighthouse

After this fix, we do not have this error.

**Overall change:** When Ares does not provide a promo.summary to use in the meta description element, we use the value from seoHeadline. 

**Code changes:**

- Fallback value added
- Cypress test updated
- Lighthouse SEO threshold value returned back to 80. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by ChrisBAshton** [comment]
Blocked on #1417, after which we should revert the Lighthouse threshold change introduced in that PR.

**Comment by ChrisBAshton** [comment]
Related to #842?

**Comment by sareh** [comment]
@ChrisBAshton I created a bug ticket that specifically focuses on this issue: #1430

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "test:e2e:storybook": "npm run killStorybook && run-p --race storybook cypress:storybook",
     "test:e2e:storybook:ci": "run-p --race storybook:ci cypress:storybook",
     "test:e2e:storybook:interactive": "npm run killStorybook && run-p --race storybook cypress:storybook:interactive",
-    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=70 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=70",
+    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=70",
```
Comment (dr3) [inline package.json]:
```suggestion
    "test:lighthouse": "lighthouse-ci http://localhost:7080/news/articles/c9rpqy7pmypo --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80 && lighthouse-ci http://localhost:7080/news/articles/c85pqyj5m2ko --performance=80 --pwa=62 --accessibility=100 --best-practices=93 --seo=80",
```

**Review by dr3 — DISMISSED** [review]

**Review by pjlee11 — DISMISSED** [review]

**Review by ChrisBAshton — DISMISSED** [review]
Nice

**Review by dr3 — DISMISSED** [review]

**Review by jamesdonoh — DISMISSED** [review]

**Review by pjlee11 — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**